tests/e2e/test_api.py

```diff
@@ -407,14 +405,15 @@ def test_valid_question() -> None:
         assert NO_RAG_CONTENT_RESP not in json_response["response"]
 
         score = ResponseValidation().get_similarity_score(
-            json_response["response"], ANSWER1
+            json_response["response"], eval_answer
         )
         assert score <= SCORE_THRESHOLD
 
 
 @pytest.mark.standalone
-def test_valid_question_tokens_counter() -> None:
+def test_valid_question_tokens_counter(qna_data) -> None:
```

Comment:
we need to keep our tests more focused.  This test is for checking the token counter metric.  It should not also be evaluating the quality of the response, let's keep that in a separate test.

Otherwise understanding the implications of a test failure (what does it mean is broken when this test fails) becomes harder, as does understanding what tests exist to test various functionality.

Reply:
> I am not evaluating response quality here, only picking a question..

ah, true.

> But I will revert to earlier hardcoded question to avoid loading file and possible error because of this.

yeah probably still a good idea not to use the eval question dataset for tests that aren't focused on the question/response.  keeps them slightly simpler.